ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -303,6 +303,8 @@ namespace ix
             }
         }
 
+        if (lastingTimeoutDelayInMs) lastingTimeoutDelayInMs = 20;
```

Comment:
That one looks like debug code (overriding to 20ms), or do you want to always timeout after 20ms on Windows ? That could solve some problems since the select interrupt is a no-op on Windows.